ramp-frontend/ramp_frontend/tests/test_ramp.py

```diff
@@ -411,6 +413,119 @@ def test_ask_for_event_mail(client_session):
                         in outbox[0].body)
 
 
+@pytest.mark.parametrize(
+    "submission_dir, file_name, correct",
+    [("submissions/error", "classifier.py", True),
+     ("submissions/random_forest_10_10", "classifier.py", True),
+     ("submissions/starting_kit", "classifier.py", True),
+     ("submissions/starting_kit", "classifier2.py", False),
+     ("/", "README.md", False),
+     ("/", "requirements.txt", False)]
+)
+def test_sandbox_upload_file(client_session, makedrop_event,
+                             submission_dir, file_name, correct):
+    client, session = client_session
+    sign_up_team(session, 'iris_test_4event', 'test_user')
+
+    config = ramp_config_template()
+    ramp_config = generate_ramp_config(read_config(config))
+
+    # upload file in sandbox.html
+    path_submissions = os.path.join(
+                                    ramp_config['ramp_kit_dir'],
+                                    submission_dir
+                                   )
+
+    with login_scope(client, 'test_user', 'test') as client:
+        rv = client.get('http://localhost/events/iris_test_4event/sandbox')
+        assert rv.status_code == 200
+
+        # choose file and check if it was uploaded correctly
+        path_submission = os.path.join(path_submissions, file_name)
+
+        try:
+            rv = client.post('http://localhost/events/'
+                             + 'iris_test_4event/sandbox',
+                             headers={'Referer':
+                                      'http://localhost/events/'
+                                      + 'iris_test_4event/sandbox'},
+                             data={'file': (open(path_submission, 'rb'),
+                                            file_name)},
+                             follow_redirects=False)
+
+            assert rv.status_code == 302
+            assert rv.location == 'http://localhost/events/' \
+                                  'iris_test_4event/sandbox'
+
+            # code of the saved file
+            with open(path_submission, 'r') as file:
+                submitted_data = file.read()
+        except FileNotFoundError:
```

Comment:
What is the reason for the FileNotFoundError?

Basically I would expect that if the file is not found, we would like to check that we raise this error instead of using `except`.